### PR TITLE
Awesome Books - Classes

### PR DESCRIPTION
**Pull request by Ursaaaaaa**
- Used JavaScript classes instead of objects and arrays.
- Maintained all of the original functionality of the application
- Added CSS styles to the application to make it match this wireframe:

### CODE REVIEW

File: books.js [hunk 1]
```diff
@@ -1,68 +1,70 @@
...
+function showBooks() {
+  const booksCode = books.map((book) => new BookClass(book.title, book.author, book.id).bookCode());
+  bookList.innerHTML = booksCode.join('');
+  const deleteBtn = document.querySelectorAll('.remove');
+  deleteBtn.forEach((el) => {
+    el.addEventListener('click', (e) => {
+      const id = e.target.getAttribute('data-id');
+      BookClass.remove(id);
+      showBooks();
+    });
```
Comment (AnselemOdims) [inline books.js]: - [x] Kindly make this function a class method for uniformity and clarity

File: books.js [hunk 2]
```diff
@@ -1,68 +1,70 @@
...
-  return books;
-}
 
-displayBooksList(getBookList());
+  static remove(id) {
+    books = books.filter((b) => b.id !== Number(id));
+    localStorage.setItem('books', JSON.stringify(books));
+  }
+}
+const storeBooks = JSON.parse(localStorage.getItem('books'));
```
Comment (AnselemOdims) [inline books.js]: - [x] At the moment, the remove button is not functioning. Nothing is being removed from the screen. Please make sure all functionalities from the last milestone are functioning properly.

**Review by AnselemOdims — CHANGES_REQUESTED** [review]
### Hi @ramrod433 and @Dinma-Faith , 
You have been doing a tremendous job so far :ok_hand:
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight 🎉
PR has a title and summary. :heavy_check_mark:
No linter errors :heavy_check_mark:

## Required Changes ♻️
- [ ] Please also check inline comments under the review

## Optional suggestions
_Comments with the [OPTIONAL] prefix are not a must to implement however, It's strongly recommended to work on them as they can make your code better._

## Happy coding!👏👏👏

#### Please leave any questions or comments in the PR if you have any doubts or unclarity! You can tag me using @AnselemOdims for a quicker response

**_Note: Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: books.js (hunk 1 above)
Comment (Ursaaaaaa) [reply books.js]: - Changed this function into a Class method

File: books.js (hunk 2 above)
Comment (Ursaaaaaa) [reply books.js]: - Tested remove button and it works perfectly

**Review by julie-ify — APPROVED** [review]
## APPROVED ✔️ ✔️

Hi @ramrod433 and @Dinma-Faith, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**